cmd/snap/cmd_delete_key_test.go

```diff
@@ -44,6 +45,11 @@ func (s *SnapKeysSuite) TestDeleteKeyNonexistent(c *C) {
 }
 
 func (s *SnapKeysSuite) TestDeleteKey(c *C) {
+	// FIXME: remove once xenials gpg2 is fixed
```

Comment:
Isn't it nice to be a distributor that can actually fix this issue?

I wonder how this affects other distros. I guess I'll find out the hard way.